css/style.css

```diff
@@ -0,0 +1,215 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.main-wrapper {
+  width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.promo-header {
+  position: absolute;
+  display: flex;  
+  height: 12rem;  
+  align-items: center;
```

Comment:
Are you sure you need it here?

Reply:
I decide this moment. When these properties are excluded, the template breaks.

Reply:
Refresh `flex` knowledge https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
ok. I'll do it right now